src/backend/web/graphql/index.js

```diff
@@ -34,49 +31,52 @@ module.exports.typeDefs = graphql`
 
   # Queries to fetch data from redis
   type Query {
-    getFeed(guid: ID!): Feed
-    getFeeds: [ID]
-    getFeedsCount: Float
-    getPost(guid: ID!): Post
+    getFeedById(id: ID!): Feed
+    getFeedByUrl(url: String!): Feed
+    getFeeds: [Feed]
+    getFeedsCount: Int
+    getPost(id: ID!): Post
     getPosts(page: Int, perPage: Int): [Post]
-    getPostsCount: Float
+    getPostsCount: Int
   }
 `;
```

Comment:
Just a comment.
The proposal in #591 aims to have fewer queries, but very flexible ones. Instead of having a query for one object and another one for a list, we can add filters to the one that returns a list. Something like:
```
{
  getPosts(page:0, perPage: 20, { filter: { author: Josue } } ) {
    title
    published
    text
    html
  }
}
```
That combined with restructuring our schema would lead to something like:
```
type Feed {
  id: String
  name: String
  url: String
  posts: [Post]    <- Array of Posts
}
```
And our queries could become more flexible and effective:
```
{
  getFeeds( { filter: { author: Josue } } ) {    <- that could be id, url...
    url
    posts {
      title
      html
      published
      ...
    }
  }
}
```

Reply:
Awesome.  What, if anything, should I be doing in this PR in anticipation of that?

Also, can you look at this line that I added:

```
getFeedByUrl(url: String!): Feed
```

I'm pretty sure that's wrong, but I don't know what it should be.  All of my changes to the graphql code feel uncertain to me, so let me know what I need to fix.

Reply:
I think we can make changes to our queries once we add filters and we start combining our types (and maybe add some new ones).
Right now, what you have here is perfect.
`getFeedByUrl` looks good to me, same logic as `getFeedById` and `getPost`.

Just tested it. It's empty but GraphQL didn't complain about the query itself
![Screenshot_20200123_210209](https://user-images.githubusercontent.com/23108901/73039270-5a6b9980-3e23-11ea-98cd-c166c2e6fd28.png)